components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -19,3 +19,173 @@ Try these steps if Chef Automate returns an error while restoring data.
+
+      ```sh
+         [object_storage.config]
+          google_service_account_file = ""
+          location = ""
+          bucket_name = ""
+          access_key = ""
+          secret_key = ""
+          endpoint = ""
+          region = ""
+      ```
+
+      - If you configured pre deployment, then we are good
+      - If you want to change the `bucket` or `base_path`, then use below template for Frontend nodes
+
+      ```sh
+      [global.v1]
+        [global.v1.external.opensearch.backup.s3]
+            bucket = "<BUCKET_NAME>"
+            base_path = "opensearch"
+         [global.v1.backups.s3.bucket]
+            name = "<BUCKET_NAME>"
+            base_path = "automate"
+      ```  
+
+      - You can choose any value for the variable `base_path`. `base_path` patch is only required for the frontend node.
+      - Use the command  to apply the above template `chef-automate config patch frontend.toml --fe`
+      - Post the configuration patch, use the curl request to validate
+
+      ```sh
+         curl localhost:10144/_snapshot?pretty
+      ```
+
+         - If the response is empty `{}`, then we are good
+
+         - If the response has json output, then it should have correct value for the `base_path`
+
+         ```sh
+         {
+             "chef-automate-es6-event-feed-service" : {
+               "type" : "s3",
+               "settings" : {
+                 "bucket" : "MY-BUCKET",
+                 "base_path" : "opensearch/automate-elasticsearch-data/chef-automate-es6-event-feed-service",
+                 "readonly" : "false",
+                 "compress" : "false"
+               }
+             },
+             "chef-automate-es6-compliance-service" : {
+               "type" : "s3",
+               "settings" : {
+                 "bucket" : "MY-BUCKET",
+                 "base_path" : "opensearch/automate-elasticsearch-data/chef-automate-es6-compliance-service",
+                 "readonly" : "false",
+                 "compress" : "false"
+               }
+             },
+             "chef-automate-es6-ingest-service" : {
+               "type" : "s3",
+               "settings" : {
+                 "bucket" : "MY-BUCKET",
+                 "base_path" : "opensearch/automate-elasticsearch-data/chef-automate-es6-ingest-service",
+                 "readonly" : "false",
+                 "compress" : "false"
+               }
+             },
+             "chef-automate-es6-automate-cs-oc-erchef" : {
+               "type" : "s3",
+               "settings" : {
+                 "bucket" : "MY-BUCKET",
+                 "base_path" : "opensearch/automate-elasticsearch-data/chef-automate-es6-automate-cs-oc-erchef",
+                 "readonly" : "false",
+                 "compress" : "false"
+               }
+             }
+         }
+         ```
+
+         - In case of `base_path` value is not matching, then we have to delete the existing `snapshot`. please refer the steps from the file system
```

Comment:
- In case of `base_path` value is not matching, then we have to delete the existing `snapshot`. please refer to the steps from the file system